src/main/java/net/minecraftforge/common/DimensionManager.java

```diff
@@ -405,7 +412,10 @@ public static void readRegistry(CompoundNBT data)
                     savedEntries.put(entry.name, entry);
                     continue;
                 }
-                registerDimensionInternal(entry.id, entry.name, mod, entry.data == null ? null : new PacketBuffer(Unpooled.wrappedBuffer(entry.data)), entry.skyLight());
+                if (mod.loadFromLevelDat()) {
+                    registerDimensionInternal(entry.id, entry.name, mod, entry.data == null ? null : new PacketBuffer(
```

Comment:
Just keep it one big line

Reply:
No, this is not a patch file...

Reply:
Ok, i reverted it

Reply:
Newline for the register statement is good, but IMO the registerDimensionInternal should be kept in one line. But that may be personal preference and not forge code style

Reply:
Yes I agree ichttt

Reply:
Ok i'll revert it to single line then